AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -291,6 +291,10 @@ - (void)_initializeInstance
   
   _environmentState = ASEnvironmentStateMakeDefault();
   
+  _defaultLayoutTransitionDuration = 0.2;
+  _defaultLayoutTransitionDelay = 0.0;
+  _defaultLayoutTransitionOptions = NULL;
```

Comment:
Let's add `kNilOptions` and `UIViewAnimationOptionBeginFromCurrentState` explicitly in here